pkg/kubelet/apis/deviceplugin/v1beta1/api.proto

```diff
@@ -48,6 +55,11 @@ service DevicePlugin {
 	// Plugin can run device specific operations and instruct Kubelet
 	// of the steps to make the Device available in the container
 	rpc Allocate(AllocateRequest) returns (AllocateResponse) {}
+
+        // PreStart is called, if indicated by Device Plugin during registeration phase,
+        // before each container start. Device plugin can run device specific operations
+        // such as reseting the device before making devices available to the container
+	rpc PreStart(PreStartRequest) returns (Empty) {}
```

Comment:
Rather than return `Empty`, can we have an empty object for now? The reason I say is that if we choose to add a PreStart `Exec` hook, one way to model that will be to return an Exec command as part of `PreStart` RPC, assuming Kubelet can invoke PreStart between CRI `Create` and `Start`.
Having an empty object makes it easy to add fields without requiring an API bump.